pyccel/ast/core.py

```diff
@@ -2281,6 +2282,44 @@ def cls_variable(self):
         """
         return self._cls_variable
 
+class KernelCall(FunctionCall):
+    """
+    Represents a kernel function call in the code.
+    # Example of a method call with array-like indexing: a[c, b]()
```

Comment:
Can we put the cuda specific objects in a new file please. These are not core concepts